tests/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -0,0 +1,497 @@
+import pytest
+import logging
+import time
+import ocpnetsplit
+
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed, CephHealthException
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.node import get_all_nodes, wait_for_nodes_status
+from ocs_ci.helpers.sanity_helpers import Sanity
+from datetime import datetime, timedelta, timezone
+from ocs_ci.ocs.resources.pvc import get_pvc_objs
+from ocs_ci.ocs.resources.pod import (
+    Pod,
+    wait_for_pods_to_be_in_statuses,
+    get_pods_having_label,
+    get_pod_logs,
+    get_pod_node,
+    get_ceph_tools_pod,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def check_for_read_pause(logreader_pods, start_time, end_time):
```

Comment:
how about creating a workload util file for logreader? The functionality could be used in other, non arbiter tests